helmfile/charts/node-local-dns/templates/node-local-dns.yaml

```diff
@@ -98,24 +92,20 @@ data:
         prometheus :9253
         }
     .:53 {
-        log {
-          class error
-        }
         errors
         cache 30
         reload
         loop
         bind {{ .Values.localIP }} {{ .Values.clusterDNS }}
-        forward . __PILLAR__UPSTREAM__SERVERS__ {
-                force_tcp
-        }
+        forward . __PILLAR__UPSTREAM__SERVERS__
```

Comment:
Is this change based on something upstream? Making changes like this deserves at least some explanation as well as a version bump of the chart.

Reply:
https://github.com/kubernetes/kubernetes/blob/master/cluster/addons/dns/nodelocaldns/nodelocaldns.yaml

Reply:
Apparently it can improve performance, https://github.com/kubernetes/kubernetes/pull/93471

Reply:
Thanks!